easybuild/easyconfigs/q/Qt6/Qt6-6.9.1-GCCcore-14.2.0.eb

```diff
@@ -0,0 +1,89 @@
+easyblock = 'CMakeNinja'
+
+name = 'Qt6'
+version = '6.9.1'
```

Comment:
There are a couple of additional bug fix releases, we should pick up on those.

I propose we bump this to version 6.9.3 (there's also 6.10.0, but that's perhaps a bridge too far).

I'll push an update in this PR to bump it to 6.9.3...